stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/internal/Context.kt

```diff
@@ -54,11 +58,31 @@ internal fun Context.copyToClipboard(text: String) {
     clipboard.setPrimaryClip(ClipData.newPlainText(null, text))
 }
 
-internal fun Context.createStreamThemeWrapper(): Context {
+internal fun Context.createStreamThemeWrapper(uiMode: UiMode = ChatUI.uiMode): Context {
     val typedValue = TypedValue()
+
     return when {
-        theme.resolveAttribute(R.attr.streamUiValidTheme, typedValue, true) -> this
-        theme.resolveAttribute(R.attr.streamUiTheme, typedValue, true) -> ContextThemeWrapper(this, typedValue.resourceId)
+        theme.resolveAttribute(R.attr.streamUiValidTheme, typedValue, true) ->
+            ContextThemeWrapper(this, R.style.StreamUiEmptyTheme)
+        theme.resolveAttribute(R.attr.streamUiTheme, typedValue, true) ->
+            ContextThemeWrapper(this, typedValue.resourceId)
         else -> ContextThemeWrapper(this, R.style.StreamUiTheme)
+    }.apply {
```

Comment:
Instead of apply the configuration over the ContextThemeWrapper, we can add it to the inital context and work with it.
Something like:
```
internal fun Context.createStreamThemeWrapper(uiMode: UiMode = ChatUI.uiMode): Context {
    val typedValue = TypedValue()
    val configuredContext = configureUiMode(uiMode)

    return when {
        configuredContext.theme.resolveAttribute(R.attr.streamUiValidTheme, typedValue, true) -> configuredContext
        configuredContext.theme.resolveAttribute(R.attr.streamUiTheme, typedValue, true) ->
            ContextThemeWrapper(configuredContext, typedValue.resourceId)
        else -> ContextThemeWrapper(configuredContext, R.style.StreamUiTheme)
    }
}

private fun Context.configureUiMode(uiMode: UiMode): Context = when (uiMode) {
    UiMode.LIGHT -> createConfigurationContext(resources.configuration.apply { this.uiMode = Configuration.UI_MODE_NIGHT_NO })
    UiMode.DARK -> createConfigurationContext(resources.configuration.apply { this.uiMode = Configuration.UI_MODE_NIGHT_YES })
    UiMode.SYSTEM -> this
}
```

Reply:
I didn't know about `Context.createConfigurationContext`. Yep, that's better.